apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+			maxReconnectAttempts
+		} = polling;
+
+		if (isOfflineMode || connectionStatus === 'offline') {
+			return (
+				<div className="flex items-center gap-2">
+					<div
+						className="flex items-center gap-1 text-red-400"
+						title="Offline mode - using cached data"
+					>
+						<div className="w-2 h-2 rounded-full bg-red-400"></div>
+						<span className="text-xs">Offline</span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)